internal/compression/gzip.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compression
+
+import _ "google.golang.org/grpc/encoding/gzip"
```

Comment:
You and I know that it is for gRPC compression but just the name of the package "compression" loses context and it could be compression for payloads. We use more than 1 serialization format, e.g. gRPC and Thrift, but also compression could be for other things. I think we should just place this code in exporterparser/opencensus.go instead of otherwise it is going to be very confusing.

Reply:
This is compression for both the collector and receiver, and therefore doesn't make sense in exporterparser. The initial plan was to have all compression be implemented here (be it for grpc, thrift, etc). But if we want to break out what is for grpc, thrift, http, etc, we could make separate packages within compression, where we would leave `types.go` where it is (to have functions for what is supported by various encodings) and then a compression/grpc package that contains the grpc specific compression, compression/thrift for thrift, etc.

Reply:
Roger that! Thanks for the explanation.